generate/definitions/enums

```diff
@@ -119,6 +119,19 @@ TransactionState int8 camelcase (
   7: PrepareEpochFence
 )
 
+////////////
+// QUOTAS //
+////////////
```

Comment:
Needs a newline after so that this section is not a part of the kmsg documentation.